devfiles/m2-repo-volume/devfile.yaml

```diff
@@ -42,21 +42,21 @@ parent:
                                                             && if [ ! -d /home/user/.m2/repository ];
                                                                  then echo "...copying cached maven dependencies" && cp -r /mvn/repository /home/user/.m2/repository;
                                                                fi
-                                                            && mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.6 package
+                                                            && mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.9 package
                                                             && touch ./.disable-bld-cmd;
                    fi
     - id: run
       exec:
-        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.6 -Ddebug=false -DhotTests=true -DcompileWait=3 liberty:dev
+        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.9 -Ddebug=false -DhotTests=true -DcompileWait=3 liberty:dev
     - id: run-test-off
       exec:
-        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.6 -Ddebug=false liberty:dev
+        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.9 -Ddebug=false liberty:dev
     - id: debug
       exec:
-        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.6 -DdebugPort=${DEBUG_PORT} liberty:dev -Dliberty.env.WLP_DEBUG_REMOTE=y
+        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository -Dliberty.runtime.version=21.0.0.9 -DdebugPort=${DEBUG_PORT} liberty:dev -Dliberty.env.WLP_DEBUG_REMOTE=y
     - id: test
       exec:
-        commandLine: mvn -Dmaven.repo.local=/home/user/.m2/repository failsafe:integration-test failsafe:verify
+        commandLine: mvn compiler:compile -Dmaven.repo.local=/home/user/.m2/repository failsafe:integration-test failsafe:verify
```

Comment:
is the compiler:compile needed to cause the testcases to re-build whenever the test cmd is invoked?

Reply:
It's needed to resolve inter-module dependencies within the top-level reactor build, if those deps aren't in the local m2.  (And with liberty:dev, the app-built deps wouldn't be in m2 necessarily)

Reply:
Kind of a Maven quirk that this resolves the problem.. can't even find it documented..but there it is.